components/SEO.js

```diff
@@ -37,11 +37,37 @@ export const BlogSeo = ({ title, summary, date, lastmod, images = [] }) => {
 
   const featuredImages = imagesArr.map((img) => {
     return {
+      '@type': 'ImageObject',
       url: `${siteMetadata.siteUrl}${img}`,
-      alt: title,
     }
   })
 
+  const structuredData = {
+    '@context': 'https://schema.org',
+    '@type': 'Article',
+    mainEntityOfPage: {
+      '@type': 'WebPage',
+      '@id': url,
+    },
+    headline: title,
+    image: featuredImages,
+    datePublished: publishedAt,
+    dateModified: modifiedAt,
+    author: {
+      '@type': 'Person',
+      name: siteMetadata.author,
```

Comment:
`author` should be a list of authors since now we support multiple authors and taken from the frontmatter. If none is provided we can fall back to `siteMetadata.author`.  To verify that it displays correctly you can try it on this page - https://tailwind-nextjs-starter-blog-7uibpj70o-timlrx.vercel.app/blog/introducing-tailwind-nextjs-starter-blog